src/main.js

```diff
@@ -33,6 +36,19 @@ if (process.env.NODE_ENV === 'production') {
     })
 }
 
+Sentry.init({
+    dsn: process.env.NODE_ENV === 'production'
+        ? 'https://8c09726e231d4cf780c541f40d3639a9@sentry.io/3009295' // cc-chooser-prod project
+        : 'https://ab63acb8c1464466869182dd53c7046d@sentry.io/3009597', // cc-chooser-dev project
+    integrations: [
+        new Integrations.Vue({
+            Vue,
+            attachProps: true,
+            logErrors: !process.env.NODE_ENV === 'production' // Only log errors in dev env
```

Comment:
Why aren't we logging errors in production?

Reply:
This flag enables/ disables error logging from the Vue renderer, from [Sentry's Vue docs](https://docs.sentry.io/platforms/javascript/vue/):
`
Passing in logErrors is optional and is false if it is not provided. If you set it to true, Sentry will call original Vue’s logError function as well.
`

Reply:
thanks